apply easy food suggestions use case

AbdoMagdey24 opened the pull request:
Task Link on the board (https://github.com/orgs/damascus-squad/projects/6/views/1?pane=issue&itemId=106426369)
- apply the use case 
- add it to the use case module for dependency injection 
- pass it to the UI class to display its result when user choose it

AbdoMagdey24 committed:
apply easy food suggestions use case

Al-Taie commented on src/main/kotlin/presentation/FoodChangeModeUI.kt:
```diff
@@ -2,9 +2,12 @@ package org.damascus.presentation
+import org.damascus.model.Meal
+import org.damascus.useCase.GetEasyFoodSuggestionsUseCase
 
 class FoodChangeMoodUI(
```
`FoodChangeMoodUI` -> `FoodChangeMoodUi`

AbdoMagdey24 replied:
Resolved

Al-Taie commented on src/main/kotlin/presentation/FoodChangeModeUI.kt:
```diff
@@ -2,9 +2,12 @@ package org.damascus.presentation
+    private val getFirstNMealsUseCase: GetFirstTenMealsUseCase,
+    private val getEasyFoodSuggestionsUseCase: GetEasyFoodSuggestionsUseCase
 ) {
     private fun getInput() = readLine()?.toIntOrNull()
```
move this to the end of file

AbdoMagdey24 replied:
Why we need to do this ?!

Asmaa7071 replied:
the private functions should be at the end of file

AbdoMagdey24 replied:
resolved @Al-Taie

AbdoMagdey24 committed:
resolve comments:
- follow naming rules and modify FoodChangeMoodUI to be FoodChangeMoodUi
- move the private getInput function to the end of the class
